CHANGELOG.md

```diff
@@ -12,6 +12,7 @@
 
 ### Bug fixes
 
+* [#7871](https://github.com/rubocop-hq/rubocop/pull/7871): Fix an auto-correction bug in Lint/BooleanSymbol. ([@knu][])
```

Comment:
Oops, I overlooked one. Can you enclose the cop name in backticks?

```suggestion
* [#7871](https://github.com/rubocop-hq/rubocop/pull/7871): Fix an auto-correction bug in `Lint/BooleanSymbol`. ([@knu][])
```

Reply:
Done!